src/application.coffee

```diff
@@ -880,7 +882,7 @@ application.initialize = ->
 	listenToEvents()
 	getAndApplyDeviceConfig()
 	.then ->
-		knex('app').whereNot(markedForDeletion: true).select()
+		knex('app').whereNot(markedForDeletion: true).orWhereNull('markedForDeletion').select()
```

Comment:
This change seems unrelated, what is it for?

Reply:
Ok, I read the commit message properly this time and it makes sense, however doesn't `whereNot..true` find cases where it's null anyway?

Reply:
Explained below in your other question - part of the second commit which fixes a different bug for preloaded apps.

Reply:
whereNot true doesn't match null values :(